Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfigurationInternal.h

```diff
@@ -50,3 +53,13 @@ _WKDragLiftDelay toDragLiftDelay(NSUInteger);
 _WKDragLiftDelay toWKDragLiftDelay(WebKit::DragLiftDelay);
 WebKit::DragLiftDelay fromWKDragLiftDelay(_WKDragLiftDelay);
 #endif
+
+#endif // __cplusplus
+
+@interface WKWebViewConfiguration (NonCpp)
+
++ (nullable NSString *)_canonicalSchemeForURLScheme:(NSString * _Nonnull)urlScheme;
```

Comment:
```suggestion
+ (nullable NSString *)_canonicalSchemeForURLScheme:(nonnull NSString *)urlScheme;
```
The reason you're being told to put nonnull is because this isn't within an audit region. Auditing the C++ parts above is maybe difficult to do right now, but I think you could probably still add the guards around just this interface.